python/ray/tune/impl/config.py

```diff
@@ -0,0 +1,46 @@
+from dataclasses import dataclass
+
+from ray.air.config import CheckpointConfig as _CheckpointConfig
+from ray.air.config import FailureConfig as _FailureConfig
+from ray.air.config import RunConfig as _RunConfig
+from ray.train.constants import _v2_migration_warnings_enabled
+from ray.train.utils import _copy_doc, log_deprecation_warning
+
+# NOTE: This is just a pass-through wrapper around `ray.train.RunConfig`
+# in order to detect whether the import module was correct (e.g. `ray.tune.RunConfig`).
+
+
+@dataclass
+@_copy_doc(_CheckpointConfig)
+class CheckpointConfig(_CheckpointConfig):
+    pass
+
+
+@dataclass
+@_copy_doc(_FailureConfig)
+class FailureConfig(_FailureConfig):
+    pass
```

Comment:
Note: even though the `__doc__` gets copied, the vscode docstrings in code-completion does not show anything. Only way to populate that is copying over the docstring. WDYT?

Reply:
We can follow-up with copying over the docstring to this one.